src/api/app/helpers/webui/packages/partials_path_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Webui::Packages::PartialsPathHelper
+  def partial_path(directory, partial)
+    File.join('webui2', 'webui', 'package', directory, partial)
```

Comment:
I don't think we need to have code for this. It's already possible to achieve this by specifying the full path to the partial starting from `app/views`. This is what this code does too

Reply:
After talking with @vpereira, I am also fine with this approach. I prefer not having an helper, but it's not a issue to have it too. I won't block this PR due to this.